shared-config/log4j.properties

```diff
@@ -17,4 +17,4 @@ log4j.appender.file.layout.ConversionPattern=%d{ISO8601} [%t] %-5p %40C - %m%n
 #package specific levels
 log4j.logger.com.amazon.dataprepper.pipeline=INFO
 log4j.logger.com.amazon.dataprepper.parser=INFO
-log4j.logger.com.amazon.dataprepper.plugin=INFO
+log4j.logger.com.amazon.dataprepper.plugins=INFO
```

Comment:
I have kept fixing this in the images i uploaded to ECR and always postponed to push a PR. Thanks for doing this

Reply:
I have also been meaning to fix this for awhile but have been pushing it off too 😉